ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/pull-queries-against-materialized-aggregates.json

```diff
@@ -1927,17 +1914,90 @@
       }
     },
     {
-      "name": "IN: fail on non-literal key",
+      "name": "non-windowed - function in where clause",
```

Comment:
Do we want to support generic expressions in pull query WHERE clauses at this time? Last I spoke to @AlanConfluent it sounded like we wanted to limit support to literal expressions (i.e., those that create arrays and structs with literal values) for now.

Reply:
it would be more work to not support this than to support this. Is there any reason why we don't want to?